src/I18n/Keys.elm

```diff
@@ -0,0 +1,6 @@
+module I18n.Keys exposing (Key(..))
+
+
+type Key
+    = IntroText
```

Comment:
Seems fine, but probably will want to call it pageTitle or something like that as it's being used in the h1. Again not too worried because when we put real content, this will change.